fastlane/Fastfile

```diff
@@ -158,10 +147,95 @@ platform :ios do
         end
     end
 
+    # SNIPPETS
+
+    def write_snippet_summary(snippets_path, file_handle)
+        file_handle.puts("# Xcode Snippets\n\n")
+        project_snippet_plists.each do |plist|
+            title = plist["IDECodeSnippetTitle"]
+            summary = plist["IDECodeSnippetSummary"]
+            prefix = plist["IDECodeSnippetCompletionPrefix"]
+            file_handle.puts("## #{title}")
+            file_handle.puts("\n")
+            file_handle.puts("#{summary}")
+            file_handle.puts("\n")
+            file_handle.puts("**Keyword: #{prefix}**")
+            file_handle.puts("\n")
+        end
+    end
+
+    def ensure_snippets_integrity(snippets_path)
+        UI.message "Checking Xcode snippets…"
+        plists = project_snippet_plists
+        keys_to_check = ["IDECodeSnippetTitle", "IDECodeSnippetIdentifier", "IDECodeSnippetSummary", "IDECodeSnippetCompletionPrefix"]
```

Comment:
I am not sure about ensuring the "IDECodeSnippetCompletionPrefix" to be non empty is a good idea. Code snippets can be easily applied by using the shortcut ⌘⇧L, then searching using work or up/down arrow and pressing enter to apply the snippet. This exactly my use case for the failing snippet on the CI

Reply:
okay, you can remove it!